bcda/auth/plugin/alpha_test.go

```diff
@@ -145,8 +145,13 @@ func (s *AlphaAuthPluginTestSuite) TestRevokeAccessToken() {
 }
 
 func (s *AlphaAuthPluginTestSuite) TestValidateAccessToken() {
-	err := s.p.ValidateAccessToken("")
-	assert.Equal(s.T(), "not yet implemented", err.Error())
+	userID := uuid.NewRandom().String()
+	acoID := uuid.NewRandom().String()
+	ts, _ := auth.InitAuthBackend().GenerateTokenString(userID, acoID)
+	err := s.p.ValidateAccessToken(ts)
+	assert.Nil(s.T(), err)
+	err = s.p.ValidateAccessToken("this.token.fails")
```

Comment:
This test should construct a structurally valid token that is invalid because 

- it is missing required claims
- the signature is invalid (because it was signed using some other key
- it was signed using the wrong algorithm

Reply:
The second 2 can be done externally on the JWT.io site and work off of a static token string.

Reply:
that's what i did!  you can't even sign it if you don't have the required claims though it seems.